setup.py

```diff
@@ -50,18 +50,30 @@
         "docutils",
         "lxml>=2.2.4",
         "plone.app.registry>=1.0",
+        "plone.base",
+        "plone.i18n",
+        "plone.memoize",
+        "plone.registry",
+        "plone.resource",
         "plone.resourceeditor>=2.0.0",
         "plone.staticresources",
         "plone.subrequest",
         "plone.transformchain",
         "python-dateutil",
+        "Products.CMFPlone",
```

Comment:
This means a circular dependency between `Products.CMFPlone` and `plone.app.theming`. :-(
It is correct, because we include `permissions.zcml` from `Products.CMFPlone.controlpanel` for the `plone.app.controlpanel.Themes` permission.
Maybe move that permission here?

cc @jensens

Reply:
Indeed, this is pitty. I think too, the clean way is to move the permission definition from CMFPlone to here. I'll prepare a PR.

Reply:
Done.

Reply:
Well, currently we need to ignore the `pull_request` test here, because there it currently ignores the branch of CMFPlone, right?

Reply:
The test will keep failing until there is a `Products.CMFPlone` release on dist.plone.org/release/6.0-dev.  That should be there before the end of the week, so let's not worry about it for now.

Thanks for moving the permission, seems a good solution.